doc/source/getting_started/install.rst

```diff
@@ -284,7 +284,7 @@ pyxlsb                    1.0.6              Reading for xlsb files
 qtpy                                         Clipboard I/O
 s3fs                      0.4.0              Amazon S3 access
 tabulate                  0.8.3              Printing in Markdown-friendly format (see `tabulate`_)
-xarray                    0.12.0             pandas-like API for N-dimensional data
+xarray                    0.12.3             pandas-like API for N-dimensional data
```

Comment:
can you update the note in 1.2 whatsnew about the min versions; also there is `pandas/compat/_optional.py`